### PR TITLE
Fixed issues with render depth in the newer react. Added babel polyfi…

### PR DESCRIPTION
**Pull request by Stefanie899**
Fixed issues with render depth in the newer react. Added babel polyfill to regular dependencies away from dev dependencies, as the object.assign polyfill is required for the Sketchfield component to work.

### CODE REVIEW

**Review by joshuapeters — APPROVED** [review]
Looks good to me! 👌

**Review by dmashuda-andculture — APPROVED** [review]

File: src/SketchField.jsx [hunk 1]
```diff
@@ -282,6 +280,28 @@ class SketchField extends PureComponent {
         } else if (canvas.getHeight() < this.props.fixCanvasToSize.height) {
             canvas.setHeight(this.props.fixCanvasToSize.height);
         }
+
+        let xTransform = canvas.viewportTransform[4] + canvas.getWidth() - prevWidth;
```
Comment (HeyKos) [inline src/SketchField.jsx]: Accessing an array via index could be sketchy. Consider checking the length of `canvas.viewportTransform` to ensure there is something there.

File: src/SketchField.jsx [hunk 2]
```diff
@@ -282,6 +280,28 @@ class SketchField extends PureComponent {
...
+        let yTransform = canvas.viewportTransform[5] + canvas.getHeight() - prevHeight;
```
Comment (HeyKos) [inline src/SketchField.jsx:285]: Same as comment on previous line.

File: src/SketchField.jsx (hunk 1 above)
Comment (HeyKos) [inline src/SketchField.jsx:284]: Accessing an array via index could be sketchy. Consider checking the length of canvas.viewportTransform to ensure there is something there.

**Pull request merged**